cmd/cluster-agent/api/v1/install.go

```diff
@@ -0,0 +1,150 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/DataDog/dd-go/log"
+	"github.com/gorilla/mux"
+)
+
+func Install(r *mux.Router) {
+	r.HandleFunc("metadata/nodes/{nodeName}/namespaces/{ns}/pods/{podName}", getPodMetadata).Methods("GET")
+	r.HandleFunc("metadata/nodes/{nodeName}", getNodeMetadata).Methods("GET")
+	r.HandleFunc("metadata", getAllMetadata).Methods("GET")
+	r.HandleFunc("{check}/events", getCheckLatestEvents).Methods("GET")
+}
+
+func getCheckLatestEvents(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	check := vars["check"]
+	supportedCheck := false
+	for _, c := range EventChecks {
+		if c == check {
+			supportedCheck = true
+			break
+		}
+	}
+	if supportedCheck {
+		w.Write([]byte("[OK] TODO"))
```

Comment:
What do you think of returns fast?
Like:
```go
	if supportedCheck {
		w.Write([]byte("[OK] TODO"))
        return
	}
	err := fmt.Errorf("[FAIL] TODO")
	log.Errorf("%s", err.Error())
	http.Error(w, err.Error(), http.StatusNotFound)
```

Reply:
Yeah I can fix that.